src/python/evaluation/common/xlsx_util.py

```diff
@@ -0,0 +1,41 @@
+import logging.config
+from pathlib import Path
+from typing import NoReturn, Union
+
+import pandas as pd
+from openpyxl import load_workbook, Workbook
+
+logger = logging.getLogger(__name__)
+
+
+def remove_sheet(workbook_path: Union[str, Path], sheet_name: str, to_raise_error=False) -> NoReturn:
+    try:
+        workbook = load_workbook(workbook_path)
+        workbook.remove(workbook[sheet_name])
+        workbook.save(workbook_path)
+
+    except KeyError as e:
+        message = f'Sheet with specified name: {sheet_name} does not exist.'
+        if to_raise_error:
+            logger.exception(message)
+            raise e
+        else:
+            logger.info(message)
+
+
+def create_and_get_workbook_path(config) -> Path:
```

Comment:
Please, add a type annotation for the config argument

Reply:
Config is a class – is it proper typing in that case? 
```
E = TypeVar('E', bound=EvaluationConfig)
def create_and_get_workbook_path(config: E) -> Path:
```